crates/python/src/qpu/isa.rs

```diff
@@ -146,19 +147,18 @@ impl PyInstructionSetArchitecture {
     }
 }
 
-#[pyfunction(client = "None")]
-#[pyo3(name = "get_instruction_set_architecture")]
-pub(crate) fn py_get_instruction_set_architecture(
-    quantum_processor_id: String,
-    client: Option<PyQcsClient>,
-) -> PyResult<PyInstructionSetArchitecture> {
-    py_sync!(async move {
+py_function_sync_async! {
+    #[pyfunction(client = "None")]
+    async fn get_instruction_set_architecture(
```

Comment:
I'd be curious to see how doc comments are handled by these macros. They should be captured as part of the `meta` rule, since `/// doc comment` is syntactic sugar for `#[doc = "doc comment"]`

Reply:
huh - it seems to work in that it applies the comment to the generated/public methods, but not to the internal source function